src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -326,6 +333,11 @@ class SettingsMenu extends Menu {
         // If closed by keypress, focus appropriate element.
         const key = normalizeKey(evt && evt.sourceEvent && evt.sourceEvent.key);
         const gearButton = this.controlbar.elements.settingsButton.element();
+        
```

Comment:
micro-optimization/nit: move this block to line 325 so we can keep the DOM changes together.